### DISCUSSION TITLE
Zwave 800 Controller Support?

### DISCUSSION THREAD

**Discussion by Botched1** [zwave-js/zwave-js#5246] [General] [answered]
Anyone know if/when Zwave 800 controllers will be supported - if they aren't already - with or without LR support?

Have one of these on order (EFR32ZG23A based), but there are a few others out there as well:
https://www.thesmartesthouse.com/collections/zooz/products/zooz-800-series-z-wave-long-range-usb-stick-zst39

**Comment by AlCalzone** [comment, marked as answer]
They should already work mostly.

My test stick had some issues with advanced features (NVM backup), but I was told this has been fixed on one of the latest firmwares.

Haven't re-tested though.

**Reply by AlCalzone** [reply]
Let's keep this all in #5257

**Comment by Botched1** [comment]
Thanks for the info! I'll try to find time over the holidays to test it out after my stick gets here.

Great work, as always.

**Comment by Botched1** [comment]
I got the usb stick so should be able to try it out soon.

It came with 7.18.1 firmware. I'm not going to try and update it, as it isn't clear to me which GBL is the right one - I'll let someone else be the guinea pig.

**Reply by kpine** [reply]
You should ask Zooz that question.

**Reply by Botched1** [reply]
> You should ask Zooz that question.

I agree, and I have. We'll see what they say. 

I mean knowing the chip used, there is only really a small number of possibilities depending on the radio, but I'm not in a guessing mood. :)

**Reply by Botched1** [reply]
Just FYI their support will provide firmware updates if asked. I know they have through 7.18.3.

**Comment by Botched1** [comment]
So it does work (screenshot below).

Any guidance on what to put for the power level and power output settings?

EDIT: I also asked Zooz support, and if they respond I'll update this. I also asked them about the LR max transmit power setting (not shown in the screenshot, as zwave js doesn't expose that parameter).

FWIW They said leave it at 0 and 0 and use +14 for the LR max transmit.


![image](https://user-images.githubusercontent.com/19985320/208131351-395d8c90-0638-4268-8ace-575aa2d80e77.png)
